api-operator/pkg/controller/api/api_controller.go

```diff
@@ -236,21 +236,25 @@ func (r *ReconcileAPI) Reconcile(request reconcile.Request) (reconcile.Result, e
 		return reconcile.Result{}, err
 	}
 
-	owner := getOwnerDetails(instance)
+	owner := k8s.NewOwnerRef(instance.TypeMeta, instance.ObjectMeta)
 	operatorOwner, ownerErr := getOperatorOwner(r)
 	if ownerErr != nil {
 		reqLogger.Info("Operator was not found in the " + wso2NameSpaceConst + " namespace. No owner will be set for the artifacts")
 	}
 	userNameSpace := instance.Namespace
 
 	//get configurations file for the controller
-	controlConf, errConf := getConfigmap(r, controllerConfName, wso2NameSpaceConst)
+	controlConf := &corev1.ConfigMap{}
+	errConf := k8s.Get(&r.client, types.NamespacedName{Namespace: wso2NameSpaceConst, Name: controllerConfName}, controlConf)
```

Comment:
is this k8s.get method is for creating and deplying configmaps ? Can't we have a more specific name like getConfigmap or deployConfigmap ?